src/corona_hakab_model/main.py

```diff
@@ -152,7 +153,7 @@ def run_simulation(args):
     print(sm)
     sm.run()
     df: pd.DataFrame = sm.dump(filename=args.output)
-    df.iloc[:, :-15].plot()
+    df.iloc[:, :-16].plot()
```

Comment:
Please put all the supervisable in one variable and use its len to choose what to plot. O/w everyone needs to change this all the time

Reply:
Fixed